tsconfig.json

```diff
@@ -1,18 +1,15 @@
 {
+  "extends": "@tsconfig/node16/tsconfig.json",
   "compileOnSave": true,
   "compilerOptions": {
-    "target": "es2018",
-    "module": "commonjs",
-    "moduleResolution": "node",
+    "strict": false,
```

Comment:
i'd prefer strict mode to be enabled. but the template didn't use strict mode so now loads of types throughout the app would be incorrect in strict mode.